src/containers/offer-page/faq-block/FaqBlock.tsx

```diff
@@ -54,7 +57,7 @@ const FaqBlock = <T extends CreateOrUpdateOfferData>({
   }
 
   const questionsAnswers = data.FAQ.map((el, idx, array) => (
-    <Box key={idx} sx={styles.faqInputsBlock}>
+    <Box key={el._id} sx={styles.faqInputsBlock}>
```

Comment:
Also can check interface for FAQ there is a `_id` property
<img width="182" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/56305508/7b5a3c67-fb04-4451-b1f1-b7a71f8f2df9">

<img width="292" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/56305508/95df105b-e5f8-42fc-9f53-845a179fe62f">

It's only for your information.
Your approach also make sense, but as you can see it make create without adding `_id: ${Date.now()} }`


https://github.com/ita-social-projects/SpaceToStudy-Client/assets/56305508/651a24cf-cbdd-4eea-96a4-21a344324fec